### PR TITLE
add option to optionset

### PR DESCRIPTION
**Pull request by ClaraRosalie**
refs #74

### CODE REVIEW

File: rdmorganiser/options/rdmo.xml [hunk 1]
```diff
@@ -1241,13 +1241,25 @@
...
+	<option dc:uri="https://rdmorganiser.github.io/terms/options/infrastructure_resources/262">
+		<uri_prefix>https://rdmorganiser.github.io/terms</uri_prefix>
+		<key>262</key>
+		<path>infrastructure_resources/262</path>
+		<dc:comment/>
+		<optionset dc:uri="https://rdmorganiser.github.io/terms/options/infrastructure_resources"/>
+		<order>2</order>
+		<text lang="en">Fast read and write of data is required (high performance storage)</text>
+		<text lang="de">Schnelles Lesen und Schreiben von Daten wird benötigt (High-Performance-Speicher)</text>
+		<text lang="fr">Les données sont lues et écrites rapidement (mémoire haute performance)</text>
```
Comment (Zack-83) [inline rdmorganiser/options/rdmo.xml:1253]:
```suggestion
		<text lang="fr">Les données nécessitent être lues et écrites rapidement (mémoire haute performance)</text>
```

**Review by Zack-83 — APPROVED** [review]

**Review by jmfrenzel — APPROVED** [review]
Ok looks fine

File: rdmorganiser/options/rdmo.xml [hunk 2]
```diff
@@ -1241,13 +1241,25 @@
...
+		<text lang="de">Schnelles Lesen und Schreiben von Daten wird benötigt (High-Performance-Speicher)</text>
+		<text lang="fr">Les données nécessitent être lues et écrites rapidement (mémoire haute performance)</text>
+		<additional_input>False</additional_input>
+	</option>
 	<option dc:uri="https://rdmorganiser.github.io/terms/options/infrastructure_resources/261">
 		<uri_prefix>https://rdmorganiser.github.io/terms</uri_prefix>
 		<key>261</key>
 		<path>infrastructure_resources/261</path>
 		<dc:comment/>
 		<optionset dc:uri="https://rdmorganiser.github.io/terms/options/infrastructure_resources"/>
```
Comment (Zack-83) [inline rdmorganiser/options/rdmo.xml:1261]:
```suggestion
			<text lang="fr">Les ressources en infrastructure de l'équipement habituel sur le lieu de travail sont suffisantes.</text>
			<additional_input>False</additional_input>
		</option>
		<option dc:uri="https://rdmorganiser.github.io/terms/options/infrastructure_resources/262">
			<uri_prefix>https://rdmorganiser.github.io/terms</uri_prefix>
			<key>262</key>
			<path>infrastructure_resources/262</path>
			<dc:comment/>
			<optionset dc:uri="https://rdmorganiser.github.io/terms/options/infrastructure_resources"/>
			<order>2</order>
			<text lang="en">Fast read and write of data is required (high performance storage)</text>
			<text lang="de">Schnelles Lesen und Schreiben von Daten wird benötigt (High-Performance-Speicher)</text>
			<text lang="fr">Les données nécessitent être lues et écrites rapidement (mémoire haute performance)</text>
			<additional_input>False</additional_input>
		</option>
		<option dc:uri="https://rdmorganiser.github.io/terms/options/infrastructure_resources/261">
			<uri_prefix>https://rdmorganiser.github.io/terms</uri_prefix>
			<key>261</key>
			<path>infrastructure_resources/261</path>
		<dc:comment/>
		<optionset dc:uri="https://rdmorganiser.github.io/terms/options/infrastructure_resources"/>
```

File: rdmorganiser/options/rdmo.xml [hunk 3]
```diff
@@ -1241,13 +1241,25 @@
...
-		<order>2</order>
+		<order>3</order>
```
Comment (Zack-83) [inline rdmorganiser/options/rdmo.xml:1262]:
```suggestion
			<order>3</order>
```

**Pull request merged**